tests/jenkins/UploadMinSnapshotsToS3.groovy

```diff
@@ -0,0 +1,50 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package jenkins.tests
+
+import org.junit.*
+import java.util.*
+import java.nio.file.*
+
+class UploadMinSnapshotsToS3 extends BuildPipelineTest {
+    private Path target;
+
+    @Before
+    void setUp() {
+        super.setUp()
+
+        binding.setVariable('WORKSPACE', 'workspace')
+
+        binding.setVariable('ARTIFACT_PROMOTION_ROLE_NAME', 'dummy_role')
+        binding.setVariable('AWS_ACCOUNT_ARTIFACT', '1234')
+        binding.setVariable('ARTIFACT_PRODUCTION_BUCKET_NAME', 'dummy_bucket')
+
+        Path source = Path.of("tests/data/opensearch-build-1.1.0.yml");
+        target = Path.of("workspace/builds/opensearch/manifest.yml");
+        Files.createDirectories(target.getParent());
+        Files.copy(source, target, StandardCopyOption.REPLACE_EXISTING);
```

Comment:
Can we have this setup pre-built since this is a test inside `tests/data` ahead of time? In which case cleanup isn't required either.

Reply:
Not sure what you mean by pre-build. It is already a part of `setUp()`

Reply:
I'll try to explain better. This code creates a folder and copies a file when `setUp` is run. I would just copy the file and commit it and remove this code.

Reply:
Got it. Removed this code. And edited the workplace variable so that it fetched from write manifest.
Thanks!

Reply:
@dblock Can you review it once before I merge?
Thanks!